app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -2,33 +2,33 @@ import React, { useCallback } from "react";
 import { SearchMap } from "components/search/SearchMap/SearchMap";
 import { SearchResult } from "components/search/SearchResults/SearchResult";
 import {
-  SearchHit,
   TransformedSearchHit,
   transformSearchResults,
 } from "models/SearchHits";
-import { useInstantSearch, useSearchBox } from "react-instantsearch";
-// eslint-disable-next-line import/no-extraneous-dependencies
-import algoliasearchHelper from "algoliasearch-helper";
+import {
+  useInstantSearch,
+  usePagination,
+  useSearchBox,
+} from "react-instantsearch";
 import styles from "./SearchResults.module.scss";
 import ClearSearchButton from "../Refinements/ClearSearchButton";
 import { Loader } from "components/ui";
 import ResultsPagination from "components/search/Pagination/ResultsPagination";
 
+export enum SearchMapActions {
+  SearchThisArea,
+}
+
 const SearchResults = ({
   mobileMapIsCollapsed,
 }: {
   mobileMapIsCollapsed: boolean;
 }) => {
+  const { refine: refinePagination } = usePagination();
   const {
+    // Results type is algoliasearchHelper.SearchResults<SearchHit>
     results: searchResults,
     status,
-  }: // uiState: { query },
-  {
-    results: algoliasearchHelper.SearchResults<SearchHit>;
-    status: "idle" | "loading" | "stalled" | "error";
```

Comment:
**note:** These types are already intuited by the language server so I felt they were redundant and could be removed.